CMakeLists.txt

```diff
@@ -214,7 +217,7 @@ endif(${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
 if (CMAKE_BUILD_TYPE STREQUAL "Debug")
   include(cmake/Modules/CodeCoverage.cmake)
   APPEND_COVERAGE_COMPILER_FLAGS()
-  if(NOT ORTOOLS_FETCH)
+  if(NOT USE_ORTOOLS_FETCH_SRC)
     target_compile_options(Fields2Cover PRIVATE -pedantic -Werror)
   endif()
```

Comment:
I think you probably are trying to find a way to treat ortools as a system library here.

This might be a possibility
https://stackoverflow.com/a/64139281

IOW - something like this might work  (I have not tried the USE_ORTOOLS_FETCH_SRC yet.)
get_target_property(ortools::ortools INTERFACE_INCLUDE_DIRECTORIES)
set_target_properties(ortools::ortools PROPERTIES INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${CATCH2_IID}")